### DISCUSSION TITLE
feature image issue

### DISCUSSION THREAD

**Discussion by azhar62355** [nasirkhan/laravel-starter#341] [General]
![9ac7b8c6-a08b-481e-b059-1bc62e291074](https://user-images.githubusercontent.com/108658167/177111220-3661e177-8e7d-462f-a2c9-f1bf46364a15.png)
![ffbc9ef6-f518-4a1a-aaba-11758e1c01d5](https://user-images.githubusercontent.com/108658167/177111230-63a2c50c-15b1-4b58-b531-ffbe4387affd.png)

**Comment by nasirkhan** [comment]
From your first screenshot I can see the first image is showing property, can you please let me know hoy did you uploaded that image ? 


Can you please answer the following queries ?
1. Have you have uploaded the image from the file browser? 
2. Are the image thumbs are visible in the file browser ? 
3. What error message it shows when you try to open the image directly using the link: http://laravel-starter.local/storage/files/blog-1.jpg?
4. Can you please share(screenshot) the physical location of the image you are trying to access and the url the application is running from. Like the file could be located at c:/wamp/www/laravel-starter/storage...

**Reply by azhar62355** [reply]
1. the first image you shown on screenshot is demo content image not uploaded from backend
2. i have uploade the image from file browser 
![Screenshot 2022-07-04 145323](https://user-images.githubusercontent.com/108658167/177125294-75b9771a-9cb4-4524-94fe-7720914d44fd.png)
![Screenshot 2022-07-04 145543](https://user-images.githubusercontent.com/108658167/177125604-175fb043-d67b-4db9-8cf6-d262751ab6c2.png)

**Reply by azhar62355** [reply]
image thumb also not visible in  browser

**Comment by nasirkhan** [comment]
From the last screenshot of the file browser I found that you are running
the application from 127.0.0.1:8000, but you are trying to view the image
from http://laravel-starter.local/...

You may create this domain from your virtual host or update the application
url from the .env file.

Nasir Khan Saikat
https://nasirkhn.com
